### ISSUE TITLE
Use Xcode 9.3b3's new xccov tool

### ISSUE THREAD

**Issue by AliSoftware** [SlatherOrg/slather#371]
Just a though for now, as I haven't even had time to even download b3 yet let alone investigate that new `xccov` and what it can do, but now that it's out it seems like it would be an interesting idea to leverage it and use it instead of custom-parsing the coverage files ourselves

![Extract of Xcode 9.3b3 Release Notes mentioning that there's a new xccov command line tool to parse Xcode coverage reports](https://pbs.twimg.com/media/DWg9mHwVAAQrEde.jpg)

Source: https://twitter.com/SmileyKeith/status/966089848904810496 (@keith)

**Comment by ksuther** [comment]
Yeah, this is definitely worth looking in to. Xcode 9.3 also appears to fix code coverage in UI tests, which will be useful to slather as well.

**Comment by LeeroyDing** [comment]
As Xcode 9.3 going public, it would be very nice to use the built-in tool for Slather.

**Comment by moming2k** [comment]
I am trying to implement that in the fork version 

https://github.com/moming2k/slather/tree/feature/add_xccov_support

It can generate the xml now. More work will need for the html report. 

I will send pull request soon

**Comment by harsh62** [comment]
@moming2k : Any updates on the implementation?

**Comment by rajeshchilukuri** [comment]
Getting error : 
xcrun: error: unable to find utility "xccov", not a developer tool or in PATH

**Comment by AliSoftware** [comment]
@rajeshchilukuri are you using a version of Xcode >9.3 ? (as in `xcode-select -p` points to 9.3+)

**Comment by rajeshchilukuri** [comment]
Yes 9.4.1

**Comment by rajeshchilukuri** [comment]
xcodebuild -version
xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

pkgutil --pkg-info=com.apple.pkg.CLTools_Executables | grep version
version: 9.4.1.0.1.1528165917

gcc --version
Configured with: --prefix=/Library/Developer/CommandLineTools/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 9.1.0 (clang-902.0.39.2)
Target: x86_64-apple-darwin17.7.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
